lua/plenary/path.lua

```diff
@@ -32,6 +32,17 @@ path.sep = (function()
   end
 end)()
 
+path.root = (function()
+    if path.sep == '/' then
+        return '/'
+    elseif path.sep == '\\' then
+        local cwd = vim.fn.cwd() -- Get the absolute path of cwd
```

Comment:
`vim.fn.cwd` doesn't exist. Its `getcwd` but `vim.loop.cwd` > `vim.fn.getcwd`.

I also think a dynamic `is_root` is better someone opened another PR like 2 hours before you opened one which has a dynamic `is_root`. See #106 (i will review it after this one we need the parent fix more for  https://github.com/nvim-telescope/telescope.nvim/pull/651)

Also can you use 2 space indents? so its consistent.

Reply:
Done

Reply:
Tests have still wrong indentation. Sorry that i am so focused on this :laughing:

Reply:
Fixed